imagebuildah/stage_executor.go

```diff
@@ -59,6 +59,7 @@ import (
 // name to the image that it produces.
 type StageExecutor struct {
 	ctx                   context.Context
+	systemContext         types.SystemContext
```

Comment:
`types.SystemContext` values in structs are customarily pointers.  This should be a pointer.